pkgs/tools/text/sgml/jade/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, pkgs, gcc, gzip, gnum4 }:
+
+stdenv.mkDerivation (rec {
+  name = "jade";
+  version = "1.2.1";
+  debpatch = "47.3";
+
+  src = fetchurl {
+    url = "ftp://ftp.jclark.com/pub/${name}/${name}-${version}.tar.gz";
+    sha256 = "84e2f8a2a87aab44f86a46b71405d4f919b219e4c73e03a83ab6c746a674b187";
+  };
+
+  patchsrc =  fetchurl {
+    url = "http://ftp.debian.org/debian/pool/main/j/jade/jade_${version}-${debpatch}.diff.gz";
+    sha256 = "8e94486898e3503308805f856a65ba5b499a6f21994151270aa743de48305464";
+  };
+
+  patches = [ patchsrc ];
+
+  enableParallelBuilding = true;
+  buildInputs = [ gcc gzip gnum4 ];
+
+  preConfigure = ''
```

Comment:
Instead: `NIX_CFLAGS_COMPILE = [ "-fpermissive" "-Wno-deprecated" ];` -- let's not use Bash when we can rely on Nix. BTW `-fpermissive` allows pretty scary things like calling undefined functions IIRC -- why is it needed?